TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Administration/UsersController.php

```diff
@@ -176,7 +183,10 @@ public function deleteAction(User $user)
             $this->addFlashMessage('You can not delete the currently logged in user', 'Current user can\'t be deleted', Message::SEVERITY_WARNING, array(), 1412374546);
             $this->redirect('index');
         }
-        $this->userService->deleteUser($user);
+        $this->securityContext->withoutAuthorizationChecks(function () use ($user) {
```

Comment:
one more thought: this may potentially open a security hole if we forget to check permissions at some other place. Sounds a bit delicate to disable security in an action controller for an operation like "delete user".